test/fixture/vcr_cassettes/actions_v1#list.json

```diff
@@ -0,0 +1,39 @@
+[
+  {
+    "request": {
+      "body": "",
+      "headers": {
+        "Accept": "application/json",
+        "Content-Type": "application/json",
+        "User-agent": "IncidentIo/elixir",
+        "Authorization": "Bearer yourtokengoeshere"
+      },
+      "method": "get",
+      "options": [],
+      "request_body": "",
+      "url": "https://api.incident.io/v1/actions?incident_id=some-incident-id&incident_mode=test&is_follow_up=false"
```

Comment:
This should match the response documented in the OpenAPI spec from https://api-docs.incident.io.